src/connection-manager/agreement.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-negated-condition, no-nested-ternary, no-bitwise -- Manipulations used for server version decoding. */
```

Comment:
Disable this for the specific lines, not for the file. Then reviewers can see exactly why this is being disabled.

Also, if google search for the error, you can get an example of allowed and disallowed code to fix linter errors